docs/description.md

```diff
@@ -1,4 +1,4 @@
 # Overview
 
-Long block description goes here.
+This block runs fastq file quality checks using FastQC. It displays the report html within Platforma, for all the files within each of the samples defined in "Samples & Data".
```

Comment:
Need to add a citation of FastQC, e.g. "if you use it in your research, please cite: [...fastqc paper...]"